packages/react/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,117 @@
+import { FC } from "react";
+//import classNames from "classnames";
+import useGlobalSettings from "../../hooks/useGlobalSettings";
+import { FooterProps } from "./Footer.props";
+//import { Image } from "../Image";
+import { LinkList } from "../LinkList";
+import { Button } from "../Button";
+
+const Footer: FC<FooterProps> = ({
+  logo,
+  tagline,
+  subtagline,
+  address,
+  linkgroup,
+  socials,
+  subscribe,
+  legal,
+  secondarylinks,
+  anchorlink,
+}) => {
+  const { prefix } = useGlobalSettings();
+
+  const baseclass = `${prefix}--footer`;
+
+  return (
+    <footer className={`${baseclass}`}>
+      <div className={`${baseclass}--main`}>
+        <div className="site--info">
+          <img
+            className={`${baseclass}--logo`}
+            src={logo}
+            alt="ILO Logo EN-White"
```

Comment:
May be just "ILO Logo". This will be read by screen readers so needs to be intelligible.